docs-team/2016-10/october-13.md

```diff
@@ -0,0 +1,101 @@
+# 2016-10-13 Agenda
+
+Previously
+
+- [ ] RM: Will talk to Mike North re: Glossary option
+- [x] TJ: Start on testing section improvements
+- [ ] RM: Learn team blog post
+  - [x] ish, started and got stumped
+- [ ] DB: Make progress on search
+- [x] LS: Bring Learn page to Face to Face to get feedback, also bring up getting help on new content such as module unification and glimmer 2.
+- [ ] Team: get RM ideas on improving website.
+- [ ] Discussion of issue labels?
+
+Topics
+
+- TJ: Remove Mirage from Tutorial? https://github.com/emberjs/guides/issues/1695
+- DB: API repo transfer?
+- DB: Help wanted update
+- RM: [ES2015 primer](https://github.com/emberjs/guides/blob/es-primer/source/localizable/getting-started/es-primer.md)
+- RM+DB: design/marketing
+
+# Action Items 
+
+- [ ] RM: Check with CLI team re: are http-mocks deprecated / going away in the long-run?
```

Comment:
More color: it's supported, not deprecated, and we test it on every release. It's an unfortunately complex portion of Ember CLI which makes it desirable to remove support, but realistically in that case it should just be moved to a core-supported addon.